packages-common/pluggable-widgets-generator/generators/app/templates/typings/jasmine.d.ts

```diff
@@ -0,0 +1,5 @@
+declare namespace jasmine {
```

Comment:
We are using jest for unit tests.. I also dont think we need it

Reply:
Removed all typings, none of them are needed.